[2025-02-23T21:50:36.305+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: kafka_spark_dag.kafka_data_stream manual__2025-02-23T21:50:27.304436+00:00 [queued]>
[2025-02-23T21:50:36.365+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: kafka_spark_dag.kafka_data_stream manual__2025-02-23T21:50:27.304436+00:00 [queued]>
[2025-02-23T21:50:36.365+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-02-23T21:50:36.415+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): kafka_data_stream> on 2025-02-23 21:50:27.304436+00:00
[2025-02-23T21:50:36.440+0000] {standard_task_runner.py:57} INFO - Started process 10206 to run task
[2025-02-23T21:50:36.466+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'kafka_spark_dag', 'kafka_data_stream', 'manual__2025-02-23T21:50:27.304436+00:00', '--job-id', '101', '--raw', '--subdir', 'DAGS_FOLDER/dag_kafka_spark.py', '--cfg-path', '/tmp/tmp2366qwc0']
[2025-02-23T21:50:36.469+0000] {standard_task_runner.py:85} INFO - Job 101: Subtask kafka_data_stream
[2025-02-23T21:50:36.705+0000] {task_command.py:416} INFO - Running <TaskInstance: kafka_spark_dag.kafka_data_stream manual__2025-02-23T21:50:27.304436+00:00 [running]> on host 5fc4543c9a07
[2025-02-23T21:50:37.333+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='kafka_spark_dag' AIRFLOW_CTX_TASK_ID='kafka_data_stream' AIRFLOW_CTX_EXECUTION_DATE='2025-02-23T21:50:27.304436+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-23T21:50:27.304436+00:00'
[2025-02-23T21:53:54.087+0000] {logging_mixin.py:154} INFO - Dataset downloaded successfully.
[2025-02-23T21:53:54.449+0000] {logging_mixin.py:154} INFO - Dataset extracted successfully.
[2025-02-23T21:53:54.449+0000] {logging_mixin.py:154} INFO - No CSV file found in extracted data.
[2025-02-23T21:53:54.453+0000] {logging_mixin.py:154} INFO - Dataset preparation failed.
[2025-02-23T21:53:54.454+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-02-23T21:53:54.546+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=kafka_spark_dag, task_id=kafka_data_stream, execution_date=20250223T215027, start_date=20250223T215036, end_date=20250223T215354
[2025-02-23T21:53:54.676+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-02-23T21:53:54.772+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
